common/lib/protocol-definitions/src/summary.ts

```diff
@@ -8,59 +8,71 @@
  *
  * @remarks
  * If any particular node is an {@link ISummaryTree}, it can contain additional `SummaryObject`s as its children.
- *
- * @public
+ * @internal
  */
 export type SummaryObject = ISummaryTree | ISummaryBlob | ISummaryHandle | ISummaryAttachment;
 
 /**
  * The root of the summary tree.
- *
- * @public
+ * @internal
  */
 export type SummaryTree = ISummaryTree | ISummaryHandle;
 
 /**
  * Type tag used to distinguish different types of nodes in a {@link ISummaryTree}.
- *
- * @public
+ * @internal
  */
 // eslint-disable-next-line @typescript-eslint/no-namespace
 export namespace SummaryType {
+	/**
+	 * @internal
+	 */
 	export type Tree = 1;
```

Comment:
Are these namespace member tags required? Seems like they should just inherit from `SummaryType`.